### PR TITLE
cleansed visitor activity types

### PR DESCRIPTION
**Pull request by jthandy**

### CODE REVIEW

**Comment by jthandy** [comment]
@drewbanin take a look. all of these are deployed in our redshift currently. am going to start working on analysis on top of them shortly.

File: models/pardot/base.sql [hunk 1]
```diff
@@ -1,8 +1,8 @@
-drop schema if exists pardot cascade;
-create schema pardot;
+drop schema if exists pardot_analysis cascade;
```
Comment (drewbanin) [inline models/pardot/base.sql]: this seems scary to me... `pardot_analysis` is definitely more uniquely identifying than `pardot`... but I can't help but think that maybe we should prefix these schemas. Something like `ac_pardot` or `__ac_pardot` ...? That would make cleaning up schemas (as mentioned below) easy as well

**Comment by drewbanin** [comment]
@jthandy  one thing noted but otherwise looks good to me

**Comment by jthandy** [comment]
I've been thinking about that those create schema commands. I don't think this is a good place for them, if they are in fact needed. I'm going to remove them and we can figure out about construction / teardown of these views later.

File: models/pardot/base.sql (hunk 1 above)
Comment (jthandy) [inline models/pardot/base.sql]: I've been thinking about that those create schema commands. I don't think this is a good place for them, if they are in fact needed. I'm going to remove them and we can figure out about construction / teardown of these views later.

**Pull request merged**